ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -99,6 +99,12 @@ function build_push_operator_images {
 
   unset GOFLAGS
   pushd ${OP_DIR}
+
+  # custom per project ENV variables
+  if [ -f .prow_ci.env ]; then
```

Comment:
we could leverage this to all other steps, but it is ok for now just to validate the proposal.